### PR TITLE
Specs update

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
update specs after rebranding

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

File: sc-meta-transactions.md [hunk 1]
```diff
@@ -2,36 +2,36 @@
...
+Furthermore, the MultiversX Standard Digital Token (ESDT) is in dire need of this functionality as well. Some users might hold only stablecoins, without any eGold to pay for the gas. Meta-transactions are thus of utmost importance for these users. Some of these users may want to pay the gas but not in eGold but in the token they hold (it depends on the relayer if it accepts payment in that token) or some relayers may want to make all the transactions free for their users (the relayer would pay the gas fee).  For example, a relayer may offer a yield farming service on MultiversX network, and may be incentivised to allow its users to quickly offer liquidity to its pools even if they do not own any eGold to pay for the actual deposit transactions.
```
Comment (schimih) [inline sc-meta-transactions.md]:
```suggestion
Furthermore, the electronic Standard Digital Token (eSDT) is in dire need of this functionality as well. Some users might hold only stablecoins, without any eGold to pay for the gas. Meta-transactions are thus of utmost importance for these users. Some of these users may want to pay the gas but not in eGold but in the token they hold (it depends on the relayer if it accepts payment in that token) or some relayers may want to make all the transactions free for their users (the relayer would pay the gas fee).  For example, a relayer may offer a yield farming service on MultiversX network, and may be incentivised to allow its users to quickly offer liquidity to its pools even if they do not own any eGold to pay for the actual deposit transactions.
```

File: sc-meta-transactions.md [hunk 2]
```diff
@@ -2,36 +2,36 @@
...
 
-The above mentioned use cases are just a few examples where meta-transactions are needed by removing the friction for users. It is up to developers to integrate this type of transactions and experiment the best onboarding experiences and different other approaches.
+The above-mentioned use cases are just a few examples where meta-transactions are needed by removing the friction for users. It is up to developers to integrate this type of transactions and experiment the best onboarding experiences and different other approaches.
 
 ## Flow example 
-* Alice wants to send some stablecoins directly to Bob: $10 as USDt. They both have accounts on Elrond blockchain but Alice does not have eGold. She has only USDT in the form of stablecoins.
+* Alice wants to send some stablecoins directly to Bob: $10 as USDt. They both have accounts on MultiversX blockchain but Alice does not have eGold. She has only USDT in the form of stablecoins.
```
Comment (schimih) [inline sc-meta-transactions.md]:
```suggestion
* Alice wants to send some stablecoins directly to Bob: $10 as USDT. They both have accounts on MultiversX blockchain but Alice does not have eGold. She has only USDT in the form of stablecoins.
```

Comment (bogdan-rosianu) [reply sc-meta-transactions.md]: done

File: sc-meta-transactions.md (hunk 1 above)
Comment (bogdan-rosianu) [reply sc-meta-transactions.md]: refactored

**Review by schimih — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**